docs/build/interoperate-qiskit-qasm3.mdx

```diff
@@ -14,7 +14,7 @@ This function is still in the exploratory phase.  Therefore, it is likely that t
 
 ## Import an OpenQASM 3 program into Qiskit
 
-Currently two high-level functions are available for importing from OpenQASM 3 into Qiskit. These functions are `load()`, which takes a file name, and `loads()`, which takes the program itself as a string: 
+Currently two high-level functions are available for importing from OpenQASM 3 into Qiskit. These functions are `load()`, which takes a file name, and `qasm3.loads()`, which takes the program itself as a string: 
```

Comment:
I think this should be either `load()` and `loads()` or `qasm3.load()` and `qasm3.loads()`. It shouldn't be `load()` and `qasm3.loads()`

Reply:
done in https://github.com/Qiskit/documentation/pull/239/commits/b05663072a19831a7587d7822abe40cbacb175a0